libcbtest/callback.c

```diff
@@ -0,0 +1,9 @@
+#include <string.h>
+
+typedef int (*callback)(const char *, int);
+
+int printString(const void *fp, const char *s) {
+    int sentinel = 10101;
```

Comment:
What is this?

Reply:
Added comment. If the FP is corrupted, the local variable on the stack will have a different value.